nifi-nar-bundles/nifi-accumulo-bundle/nifi-accumulo-processors/src/main/java/org/apache/nifi/accumulo/processors/ScanAccumulo.java

```diff
@@ -254,7 +258,12 @@ public void process(final InputStream in, final OutputStream out) throws IOExcep
 
                             try{
                                 final RecordSchema writeSchema = writerFactory.getSchema(flowAttributes, new KeySchema());
-                                try (final RecordSetWriter writer = writerFactory.createWriter(getLogger(), writeSchema, out)) {
+                                List<RecordField> fieldList = new ArrayList<>();
```

Comment:
Recommend declaring this variable and other variables as `final` following the general pattern of the Processor.  Renaming the variable to something like `writeSchemaFields` or `recordSchemaFields` would also be helpful.

Reply:
renamed variable to recordSchemaFields